bioneuron_oracle/bio_simulator.py

```diff
@@ -9,7 +9,7 @@
 class BioSimulator(NengoSimulator):
 
     def __exit__(self, *args, **kwargs):
-        for key, value in self.data.items():
+        for key, value in self.data.iteritems():
```

Comment:
Reminder to self: override [close()](https://github.com/nengo/nengo/blob/cae0266e2b6a1b774632b6cddf6ae65d88e81c3a/nengo/simulator.py#L211) rathern than `__exit__`.